cantools/database/can/formats/dbc.py

```diff
@@ -1675,7 +1677,7 @@ def _load_bus(attributes, comments):
     try:
         bus_name = attributes['database']['DBName'].value
     except KeyError:
-        return None
+        bus_name = ''
```

Comment:
not sure if empty bus names are valid...

Reply:
Yeah, this is my main concern as well. I'm sure `None` is more-invalid, so hopefully juleq can chime in with better detail.

Other options could be a dummy bus name "bus" or somehow bring forward the DBC filename to use as CANdb++ does.